### PR TITLE
Details popup

### PR DESCRIPTION
**Pull request by ashmalzahra**
This project requires the **Popup section** to be built into the Portfolio project using **Javascript objects**, which is created through pair programming.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -233,6 +233,178 @@ div p {
...
+  }
+
+  .img_description {
+    justify-content: center;
+  }
+
+  .desktop-version {
+    display: none;
+  }
+}
```
Comment (Abenezer-Tilahun) [inline style.css:266]: - [x] As stated in the project requirement Kindly make sure that the popup is responsive, it fits mobile and desktop sizes properly.So Kindly fix the responsiveness for the mobile screen size.
- Your Project Pop-up for mobile screen size Look likes this : 

Capture

-  The Figma Pop-up for Mobile Screen size Look likes this : 

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @ashmalzahra  , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git-Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️

- [x] Kindly I recommend you to update your linters.yml file with the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) linters.yml file.

https://github.com/ashmalzahra/Portfolio/blob/674146f4e70d80c1fb97663ebe7a8f76f32bc0f2/.github/workflows/linters.yml#L1-L56

_Check the comments below the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (ashmalzahra) [reply style.css:266]: Hi, thank you for pointing it out. I was using the Dimensions of iPhone SE which is why there were no problems on my end. To resolve this, I have added the overflow property to my CSS file

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+//   Title.textContent = projects.title;
+//   Description.textContent = projects.description;
+//   Tech.innerHTML = projects.technologies.map((technology) => `<li>${technology}</li>`).join('');
+//   Live.forEach((link) => {
+//     link.href = projects.links.liveVersion;
+//   });
+//   Source.forEach((link) => {
+//     link.href = projects.links.sourceCode;
+//   });
+// };
```
Comment (amejid) [inline popup.js:177]: - [ ] [OPTIONAL] I highly suggest you remove unused commented code from the codebase to make our work more clean and professional. 😎️

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**